linux_os/guide/system/network/network-nftables/package_nftables_removed/rule.yml

```diff
@@ -14,6 +14,8 @@ rationale: |-
 
 severity: medium
 
+platform: package[nftables]
+
```

Comment:
```suggestion
```
For simplicity, I would remove this line. The report will show `pass` or `fail` but not `not applicable`. It is the case for almost all (if not all) package removal rules.